src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts

```diff
@@ -1701,7 +1701,7 @@ export class VolumesListTableConfig {
 
   getTimestamp(): string {
     const dateTime = new Date();
-    return moment(dateTime).format('YYYY-MM-DD_HH-mm');
+    return format(dateTime, 'YYYY-MM-DD_HH-mm');
```

Comment:
This line doesn't work locally. 
`ERROR RangeError: Use `yyyy` instead of `YYYY` for formatting years`